src/main/java/com/github/dikhan/pagerduty/client/events/PagerDutyEventsClient.java

```diff
@@ -94,6 +96,12 @@ public EventResult resolve(ResolveIncident resolve) throws NotifyEventException
         return eventResult;
     }
 
+    public EventResult sendEvent(Incident incident) throws NotifyEventException {
```

Comment:
Agreed, though if I positioned myself as a user of the lib I'd prefer a method called explicitly with the type that should be passed in rather than the interface which then the user would have to guess or know beforehand what classes meet the contract of Incident...it's just more convenient for the user...it's true that it helps reducing the code but sometimes having a bit more code can help greatly users interact with the code more seamlessly.